packages/core/src/config/index.ts

```diff
@@ -415,7 +415,9 @@ export async function normalizeUserCompilationConfig(
     const publicPath = resolvedCompilation.output.publicPath;
```

Comment:
The problem lies here. The public path needs to be formatted. Directly modify this method and wrap it with the getValidPublicPath method.